docs/reference/announcements/870.md

```diff
@@ -50,7 +50,7 @@ With the 8.7 release, Camunda announces the deprecation of several [Zeebe gRPC](
 
 ### Removed: Tasklist GraphQL API
 
-With the 8.7 release, the deprecated [Tasklist GraphQL API](/apis-tools/tasklist-api/tasklist-api-overview.md) will be removed from the product.
+With the 8.7 release, the deprecated Tasklist GraphQL API will be removed from the product.
```

Comment:
nit, non-blocking: This doc is in version 8.7, should it read "_has been_ removed"?

Reply:
Yes! Thank you!